android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUILabel.java

```diff
@@ -51,8 +56,12 @@
 	private float shadowX = 0f;
 	private float shadowY = 0f;
 	private int shadowColor = Color.TRANSPARENT;
-	private String minimumFontSize = null;
-	private String autoshrinkSetFontSize = null;
+	private int autoLinkFlags;
+	private int viewHeightInLines;
+	private int maxLines = Integer.MAX_VALUE;
+	private float minimumFontSizeInPixels = -1.0f;
```

Comment:
Right. The "minimumFontSize" property still accepts a number, but its units is now relative to the app's default units (making it relative to all other font property units) which can be changed via "tiapp.xml" setting "ti.ui.defaultunit". That's one of the bugs I fixed, because it used to be hard-coded to pixels.

Take the below code as an example. In 6.1.0, the "font" property would have been set to "20dp" and the "minimumFontSize" would have been set to "20px". This is confusing and inconsistent. My new changes have them use the same default units.

```java
textField.font = { fontSize: 20 };
textField.minimumFontSize = 20;
```

Or perhaps you're referring to not allowing the "minimumFontSize" property to accept fontSize style strings like "20px"? You're right in that it would accept this the way it is coded now, but I was going to leave it in as an undocumented feature. But I can change it.